### ISSUE TITLE
Ingest API (Part 3)

### ISSUE THREAD

**Issue by adelefelicia** [indykite/jarvis-sdk-python#76]
**Created by: @pepe-zamarripa Aug 02 2022 at 14:39 CET**
Last updated: Aug 02 2022 at 14:41 CET
ClickUp ID: IK-3706

**Given**
The GO SDK Ingest API

**When**
Customer wants to ingest data 

**Then**
A python SDK ingest API should be available. 
Please update the wiki when feature complete: https://www.notion.so/indykite/SDK-feature-comparison-8d8333a604fc4327b0a9c14ad02f42e8
The GO interface can be found here: https://buf.build/indykite/indykiteapis. 
Use the following command to generate the required proto bufs: 
```
https://github.com/indykite/jarvis-sdk-python/blob/master/gen_proto.sh
echo "Generating protobufs ..."
buf generate buf.build/indykite/indykiteapis
buf generate buf.build/envoyproxy/protoc-gen-validate

echo "Rewriting the imports..."
packages=("indykite" "validate")

for package in  $ {packages[@]}; do
        find jarvis_sdk/indykite/. -name '*.py' -exec sed -i -e "s/from  $ {package}/from jarvis_sdk. $ {package}/g" {} \;
done

echo "Done"

```

:link: Linked tasks: 
https://github.com/indykite/jarvis-sdk-node/issues/130 ,
:warning: Waiting on: 
https://github.com/indykite/jarvis-proto/issues/1507 ,

**Comment by adelefelicia** [comment]
**Commented by: @pepe-zamarripa Aug 02 2022 at 14:41 CET**

@martasd when you come back from vacations, we need to update the title and the content of the ticket, so we can differentiate the scope during each sprint. cc @bayegan

**Comment by bayegan** [comment]
Closing - duplicate of [https://github.com/indykite/jarvis-sdk-python/issues/73](https://github.com/indykite/jarvis-sdk-python/issues/73)

**Comment by martasd** [comment]
Hi @pepe-zamarripa. Why did you reopen this issue?

**Comment by pepe-zamarripa** [comment]
Hi @martasd! I re-opened the issue because its status is blocked.
